tests/Unit/DomBuilder/CustomerDirectDebitTransferDomBuilderTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
```

Comment:
Nice, I like additional tests, really appreciate contributions like this, thank you-